usr/src/uts/aarch64/promif/prom_node.c

```diff
@@ -462,22 +451,20 @@ prom_pathname(char *buf)
 	/* nothing, just to get consconfig_dacf to compile */
 }
 
+/*
+ * `prom_init' for devicetree systems expects that a valid non-NULL DTB is
+ * passed in the `cookie' argument. The code should panic if this is not the
```

Comment:
I'm not sure what this is meaning.  Is that a later part of this stack that can't move sooner?

Reply:
I'll clean this up. We could, in the future, disentangle boot services init, shared boot code init and prom_init to the point where we could panic on a bad FDT, but this is already done in dboot.